pytest/tests/sanity/state_sync_then_catchup.py

```diff
@@ -174,6 +174,8 @@ def main():
                          1,
                          boot_node=boot_node)
     logger.info('started node1')
+    # State sync makes the storage look inconsistent.
+    node1.stop_checking_store()
```

Comment:
Would it make sense to re-enable it at the end of the test or do a one off check? 
ditto at other usages of this method

Reply:
Until it hits the GC horizon, it won't appear normal.